lore/settings.py

```diff
@@ -353,4 +353,10 @@ def get_var(name, default):
     }
 }
 HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
-ALLOW_CACHING = get_var("ALLOW_CACHING", get_var("ALLOW_CACHING", False))
+
+CACHES = {
+    "default": {
```

Comment:
So as suggested in stand up, I think it would be more flexible if we use a dedicated cache instead of the default for this in case we find need to have a more general use redis or memcached based cache later.